include/etl/placement_new.h

```diff
@@ -37,7 +37,7 @@ SOFTWARE.
 // Figure out if we can use the standard library <new> header, if haven't already done so in etl_profile.h
 #if !defined(ETL_USING_STD_NEW)
   #if defined(__has_include)
-    #define ETL_USING_STD_NEW __has_include(<new>)
+    #define ETL_USING_STD_NEW __has_include("new")
```

Comment:
As I understand [cppreference.com on source file inclusion](https://en.cppreference.com/w/cpp/preprocessor/include) it should not matter with `new` whether `""` or `<>` is used.

In order to provide for all contingencies a *compiler* switch could be introduced. I assume AVR-GCC does behave correctly for `__has_include` thus it would not need the workaround.
```suggestion
    #if defined(__ICCAVR__)
      #define ETL_USING_STD_NEW __has_include("new")
    #else
      #define ETL_USING_STD_NEW __has_include(<new>)
    #endif
```
`__ICCAVR__` is an integer that is set to 1 when the code is compiled with the IAR C/C++ Compiler for AVR.